### DISCUSSION TITLE
Weekly Hours view / Warning to Save

### DISCUSSION THREAD

**Discussion by CPC67128** [kimai/kimai#4939] [General] [answered]
Hello,
I use Kimai for 1 month roughly, it seems to fit my need. Good job and product overall (I don't use several features like Activity but the tool can be used even with my simple need).
Only thing that I feel could be interresting for me is on Weekly Hours view, I sometime enter values and then click on another view.
This doesn't save inputs by default, and it could be interresting to have a warning message box "Hey, you haven't saved your changes, do you want to continue?". Auto save is also an option.
Thanks
Steve

**Comment by kevinpapst** [comment, marked as answer]
Vote here: https://github.com/kimai/kimai/issues/3994

**Comment by CPC67128** [comment]
Than you @kevinpapst
